src/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Finals2000A.cpp

```diff
@@ -450,7 +442,8 @@ Finals2000A Finals2000A::Load(const fs::File& aFile)
 }
 
 Finals2000A::Finals2000A()
```

Comment:
can you move the constructor to the top of the file and make sure the method order follows our pattern?

Reply:
I might just not be savvy here, but do we have an established pattern? 

For this function, the constructor is private, and all object construction is done through the `Load` factory function. This is a pattern that exists in the Manager classes, too. In these cases I think it's appropriate to have the constructor definition on the bottom since it matches the header file.

Reply:
Ah yes sorry I read the file too quick... I thought it was a public constructor so I was wondering why it was at the bottom, but I see the factory patter now. Cool!